components/formats-gpl/src/loci/formats/in/GatanReader.java

```diff
@@ -378,7 +378,7 @@ else if (n == 3) {
             }
             else {
               if (dataType == 10) in.skipBytes(length);
-              else value = DataTools.stripString(in.readString(length * 2));
+              else value = in.readString(length * 2);
```

Comment:
That's the point I wanted to ask you actually since I did not want to affect other reader. 
I will review that section of the code.